src/components/Game.js

```diff
@@ -48,11 +51,27 @@ const Game = ({
   const [inputValue, setValue] = React.useState("");
   const [selectedValue, setSelectedValue] = React.useState(null);
   const [isShowConfetti, setIsShowConfetti] = React.useState(null);
+  const [currentUserStats, setCurrentUserStats] = React.useState(null);
+  const [currentUserId, setCurrentUserId] = React.useState("");
   const gameFinished = useMemo(() => isGameDone(gameStatus), [gameStatus]);
   const statsModalTimeOut = 2000;
   const handleInputChange = (value) => {
     setValue(value);
   };
+
+  useEffect(async () => {
+    return onAuthStateChanged(auth, async (user) => {
+      if (user != null) {
+        setCurrentUserId(user.uid);
+        const statsRef = doc(db, Collections.STATS_COLLECTION, user.uid);
```

Comment:
this doesnt require await?

Reply:
Not to get the reference. To get the data we need to. Added in #67.